hms-app/src/components/input-forms/HackathonForm.tsx

```diff
@@ -121,7 +135,7 @@ function HackathonForm(props: IProps) {
               label={'End Date'}
               value={endDateValue}
               onChange={setEndDateValue}
-              excludeDate={(date) => date < startDateValue!}
+              excludeDate={(date) => date < today}
```

Comment:
this disables the possibity to create a hackathon that happens only today for spontaneous ones

Reply:
The backend does not allow this anyway atm